src/Components/Sidebar/Sidebar.js

```diff
@@ -0,0 +1,70 @@
+import "./Sidebar.css";
+import Item from "../Item/Item";
+import InfoIcon from "@mui/icons-material/Info";
+import LanguageIcon from "@mui/icons-material/Language";
+import LocationOnIcon from "@mui/icons-material/LocationOn";
+import ArrowForwardIosIcon from "@mui/icons-material/ArrowForwardIos";
+import ArrowBackIosNewIcon from "@mui/icons-material/ArrowBackIosNew";
+import { useState } from "react";
+
+const Sidebar = () => {
+  const [isCollapsed, setisCollapsed] = useState(false);
+  const items = [
+    { icon: <LanguageIcon />, itemName: "All Countries" },
+    { icon: <LocationOnIcon />, itemName: "My country" },
+    { icon: <InfoIcon />, itemName: "General Info" },
+  ];
+  const onlyIcons = [
```

Comment:
You may use items for both collapsed and not collapsed state. Just need to hide the text. Since there is a repition there, you may remove this one and handle the item text accordingly.